holoviews/plotting/plotly/chart.py

```diff
@@ -255,7 +255,7 @@ def get_data(self, element, ranges, style, **kwargs):
                 values.append(sel.iloc[0, 1] if len(sel) else 0)
             bars.append({
                 'orientation': orientation, 'showlegend': False,
-                x: [xdim.pprint_value(v) for v in xvals],
+                x: xvals,
```

Comment:
Why is this not needed anymore?

Reply:
E.g.

```
import pandas as pd
import holoviews as hv
hv.extension("plotly")

df = pd.DataFrame({
    "x": [0, 1, 100],
    "y": [3, 4, 50]
})
hv.Bars(df, ["x"], ["y"])

```
<img width="344" alt="image" src="https://github.com/holoviz/holoviews/assets/15331990/3eae8cb8-ba81-49c5-9ffe-a5f369f7bbf1">


vs
<img width="730" alt="image" src="https://github.com/holoviz/holoviews/assets/15331990/51a39fe2-36f4-4762-bd5f-e2094b5d6317">